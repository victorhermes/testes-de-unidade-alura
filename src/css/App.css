h1,
p {
  margin: 0;
}

hr {
  height: 1px;
  border: 0;
  border-top: 1px solid #cecece;
  width: 100%;
  max-width: 500px;
}

.App {
  display: flex;
  flex-direction: column;
  align-items: center;
}

a {
  cursor: pointer;
}
