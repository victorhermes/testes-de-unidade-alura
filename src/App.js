import React from "react";
import { Router } from "react-router-dom";
import Routes from "./routes/routes";
import history from "./routes/history";
import "./css/App.css";

function App() {
    return (
        <Router history={history}>
            <div className="app">
                <Routes />
            </div>
        </Router>
    );
}

export default App;
