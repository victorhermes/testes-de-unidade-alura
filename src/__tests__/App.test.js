import { shallow } from "enzyme";
import React from "react";

import App from "../App";

describe("Testes do App.js", () => {
    it("Deve renderizar o componente App", () => {
        const componente = shallow(<App />);

        expect(componente.exists()).toBe(true);
        expect(componente).toMatchSnapshot();
    });
});
