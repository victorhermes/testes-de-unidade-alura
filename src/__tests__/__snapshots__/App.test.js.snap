// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testes do App.js Deve renderizar o componente App 1`] = `ShallowWrapper {}`;
